grr/server/grr_response_server/gui/ui/components/client/status_chip.ts

```diff
@@ -0,0 +1,44 @@
+import {Component, Input, ChangeDetectionStrategy, OnChanges, SimpleChanges} from '@angular/core';
+import {DateTime} from 'luxon';
+import {interval, Subject, merge} from 'rxjs';
+import {map} from 'rxjs/operators';
+
+/**
+ * Component displaying the status of a Client in a material chip.
+ */
+@Component({
+  selector: 'status-chip',
+  templateUrl: './status_chip.ng.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class StatusChip implements OnChanges {
+  private static readonly STATUS_OFFLINE = 'offline';
+  private static readonly STATUS_ONLINE = 'online';
+  private static readonly ONLINE_TRESHOLD_MINUTES = 15;
+
+  @Input() lastSeen?: Date;
+  private lastSeenChange$ = new Subject<void>();
+
+  // status observable that updates every second and when lastSeen changes
+  status$ = merge(interval(1000), this.lastSeenChange$).pipe(
+    map(_ => this.getStatus())
```

Comment:
`_` is unused. Instead, define a lambda function without arguments: `() => this.getStatus()`.